src/random.h

```diff
@@ -25,6 +26,29 @@ int GetRandInt(int nMax);
 uint256 GetRandHash();
 
 /**
+ * Rearranges the elements in the range [first,first+len) randomly, assuming
```

Comment:
Hmm, ok, maybe it's a C++ thing to use interval notation in documentation.  I can't recall seeing this style for other languages.